knowledge_runtime/knowledge_runtime/services/resolver.py

```
@@ -0,0 +1,441 @@
+
+        Args:
+            value: The value to decrypt.
+
+        Returns:
+            Decrypted value or original if not encrypted.
+        """
+        if not value:
+            return value
+        try:
+            return decrypt_api_key(value)
+        except Exception:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)